packages/visage/src/components/Fixed.ts

```diff
@@ -0,0 +1,9 @@
+import { createComponent } from '../core';
+import { Box } from './Box';
+
+export const Fixed = createComponent(Box, {
```

Comment:
I think a `div` element should be used as underlying component, there is no need to base everything on `Box` don't you think?